CHANGELOG.md

```diff
@@ -90,6 +90,7 @@ This release extends the planned support of the collections to OneView REST API
 - oneview_storage_volume_attachment_facts
 - oneview_storage_volume_template
 - oneview_storage_volume_template_facts
+- oneview_task
```

Comment:
if new endpoints are added for the task resource, then please mention them separately like below:

Added support for a new endpoint in oneview_logical_interconnect.
POST /rest/logical-interconnects/bulk-inconsistency-validation